### PR TITLE
History Api List Operations Composed

### PR DESCRIPTION
**Pull request by iammatis**
Issue #631

**Note:** There is no API endpoint to get all transaction details, therefore for every transaction one request is made. This data is used to add expiration time to composed operation model.

https://github.com/DECENTfoundation/DCorePHP-SDK/blob/43c199d27a4b1b226d5d7067d47611666e14021f/src/Sdk/HistoryApi.php#L76-L78

### CODE REVIEW

File: src/Sdk/HistoryApi.php [hunk 1]
```diff
@@ -32,6 +36,89 @@ public function listOperations(
         return $this->dcoreApi->requestWebsocket(History::class, new GetAccountHistory($accountId, $startId, $endId, $limit)) ?: [];
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function listTransferOperationsComposed(
```
Comment (petervanderkadecent) [inline src/Sdk/HistoryApi.php]: Please rename to findAllTransfersComposed to align with JS SDK

File: src/Sdk/HistoryApi.php [hunk 2]
```diff
@@ -32,6 +36,89 @@ public function listOperations(
...
+        ChainObject $accountId,
+        string $startId = '0.0.0',
+        string $endId = '0.0.0',
```
Comment (petervanderkadecent) [inline src/Sdk/HistoryApi.php]: Is endId optional? In paging client should sent only startId and limit.

File: src/Sdk/HistoryApi.php (hunk 1 above)
Comment (iammatis) [reply src/Sdk/HistoryApi.php]: Renamed to `findAllTransfersComposed `

File: src/Sdk/HistoryApi.php (hunk 2 above)
Comment (iammatis) [reply src/Sdk/HistoryApi.php]: Yes it is optional, can be dropped all together if required.

Comment (petervanderkadecent) [reply src/Sdk/HistoryApi.php]: Optional can stay.

**Pull request merged**